### DISCUSSION TITLE
Enlarge compass by entering the radius

### DISCUSSION THREAD

**Discussion by cgloeckner** [xournalpp/xournalpp#5944] [Q&A]
Hi, I love that the compass can be adjusted with a touch gesture. If no touch is available, `S` and `Shift`+`S` (or `Alt`+...) help to increase by (smaller) increments. But is there a way to enter (e.g. via UI widget) a custom radius without adjusting it "piece by piece by hand"?

glocke

**Comment by rolandlo** [comment]
It's currently not possible. If you have a good idea for UI allowing for entering the radius let us know.
It wouldn't be hard though to allow setting the radius and position of the geometry tools through the plugin system.

**Reply by cgloeckner** [reply]
The compass has a ruler which tells us the radius. Adding a small :pencil2: icon above that ruler line, that leads to a small widget (input field, apply button, cancel button) could be a solution.

Initially, I thought the cyan-colored and circled rotation label (on both compass and triangle) was to enter the rotation by number (which isn't the case). But that idea drove the desire for entering a custom radius.

About the plugin system: I haven't taken a look to it yet, I've been discovering xournal++ lately, grabbed the AppImage and literally went to work with it :smile: 

glocke
